register_confirm.php

```diff
@@ -0,0 +1,46 @@
+<?php
+require "header.php";
+?>
+<?php
+$question = $_POST['question'];
+$answer = $_POST['answer'];
+?>
+<?php
+if($question == null || $answer == null){
+    header('Location: http://localhost/FirstPhp/register.php');
```

Comment:
```suggestion
    header('Location: http://localhost/FirstPhp/register.php?message=題と答えを入力してください。');
```
header使って画面遷移する場合はこんな感じでgetでパラメータを渡して
register.php側で$message = $_GET['message'];でエラーメッセージを取得して表示してあげるのが良いかと思います。